### PR TITLE
chore: add changed_on_delta_humanized field on dashboard schema

### PR DESCRIPTION
**Pull request by zhaoyongjie**
### SUMMARY

added `changed_on_delta_humanized` field on dashboard schema. ensure rest API has this response data.

related PR: https://github.com/apache/superset/pull/15186

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/dashboards/schemas.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ class DashboardGetResponseSchema(Schema):
     owners = fields.List(fields.Nested(UserSchema))
     roles = fields.List(fields.Nested(RolesSchema))
     table_names = fields.String()  # legacy nonsense
+    changed_on_humanized = fields.String(data_key="changed_on_delta_humanized")
```
Comment (zhaoyongjie) [inline superset/dashboards/schemas.py:162]: 1. `superset.models.helps.AuditMixinNullable.changed_on_delta_humanized` is a instance method, unable to serializes.
2. ensure DashboardRestApi list_column and dashboard metadata are the same

**Review by fintzd — APPROVED** [review]
looks good to me!

**Comment by stephenLYZ** [comment]
LGTM!

File: superset/dashboards/schemas.py (hunk 1 above)
Comment (dpgaspar) [reply superset/dashboards/schemas.py:162]: to ensure 2. we should add show_columns = list_columns on the REST API class, and follow the same existing pattern

**Review by suddjian — APPROVED** [review]

File: superset/dashboards/schemas.py (hunk 1 above)
Comment (zhaoyongjie) [reply superset/dashboards/schemas.py:162]: I will follow this PR, thanks, Daniel.

**Pull request merged**